### PR TITLE
Adds HejDTO Users

### PR DESCRIPTION
**Pull request by M4c3j**

### CODE REVIEW

File: App/HejDTO/Sources/HejDTO/Users/GET/Users.Blocks.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        private enum CodingKeys: String, CodingKey {
+            case page, limit, pages, total
+            case links = "_links"
+            case embedded = "_embedded"
+        }
+    }
+}
+
+/*
+ /Users/maciejlipiec/Library/Mobile Documents/com~apple~CloudDocs/studia/Swift kursy/HejtoBuzz/App/HejDTO/Tests/HejDTOTests/Users/Users.Achievements.Tests.swift:49: error: -[HejDTOTests.UsersAchievementsTests test_parsingJsonExample] : failed: caught error: "keyNotFound(CodingKeys(stringValue: "links", intValue: nil), Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "_embedded", intValue: nil), CodingKeys(stringValue: "items", intValue: nil), _JSONKey(stringValue: "Index 0", intValue: 0), CodingKeys(stringValue: "achievement", intValue: nil)], debugDescription: "No value associated with key CodingKeys(stringValue: \"links\", intValue: nil) (\"links\").", underlyingError: nil))"
```
Comment (sloik) [inline App/HejDTO/Sources/HejDTO/Users/GET/Users.Blocks.swift:65]: what is this?

File: App/HejDTO/Sources/HejDTO/_Common/Common.Self.Link.swift [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+import Foundation
+
+extension Common {
+    
+    public struct SelfLink: Codable, Equatable {
```
Comment (sloik) [inline App/HejDTO/Sources/HejDTO/_Common/Common.Self.Link.swift:6]: Could you check in what other places you can reuse this type? :)

**Review by sloik — APPROVED** [review]
Nice work :) 

I had approved it but I do want that you handle those comments :)

File: App/HejDTO/Sources/HejDTO/_Common/Common.Self.Link.swift (hunk 2 above)
Comment (M4c3j) [reply App/HejDTO/Sources/HejDTO/_Common/Common.Self.Link.swift:6]: Of course, I will fix it tomorrow:)

File: App/HejDTO/Sources/HejDTO/Users/GET/Users.Blocks.swift (hunk 1 above)
Comment (M4c3j) [reply App/HejDTO/Sources/HejDTO/Users/GET/Users.Blocks.swift:65]: Ups.. i forgot.

**Pull request merged**